src/pyenphase/updaters/production.py

```diff
@@ -16,6 +16,9 @@ class EnvoyProductionUpdater(EnvoyUpdater):
 
     end_point = URL_PRODUCTION
     allow_inverters_fallback = False
+    working_endpoints: list[
+        str
+    ] = []  # list of successful production endpoint to use by inverters fallback
```

Comment:
If they have multiple systems this will likely cause a problem since the list isn't created in `__init__`

Reply:
Thanks for cathing this one. Tested it, and indeed it showed up in the second device. Can't add it to `__init__` as it would reset it for the 2 subclasses as well and defeat the purpose. Changed code to reset the list in the fallback class when allow_inverters_fallback is true. That solved the issue.

And it also solved my problem with --snapshot-update saving /production.json rather then /production.

Reply:
If you have two unique systems setting up at the same time it's likely still a problem since one might clear the list out from under the other one

Reply:
I think you need to move this out of this class and into the main class to track it

Reply:
My first thought would be to use common_properties pending in PR #108 to store this so it would be in the envoy data class and avoid the issue.